### ISSUE TITLE
    firefox-csshacks/chrome  /tabs_on_bottom.css

### ISSUE THREAD

**Issue by mototroyce** [MrOtherGuy/firefox-csshacks#346]
Now we're getting somewhere. However - see image - the above 'tabs_on_bottom.css' script places the MENU bar in the wrong place, causing overlap onto the browser frame controls.
![Image21](https://github.com/MrOtherGuy/firefox-csshacks/assets/140754243/e0e4cc49-844a-4a3e-8c17-c0112966706e)

**Comment by MrOtherGuy** [comment]
You also need [window_control_placeholder_support.css](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/window_control_placeholder_support.css) and then [tabs_on_bottom_menubar_on_top_patch.css](https://github.com/MrOtherGuy/firefox-csshacks/tree/master/chrome/tabs_on_bottom_menubar_on_top_patch.css) if you need to show menubar at the top.

**Comment by mototroyce** [comment]
Should I include 'window_control_placeholder_support.css' if everything seems to be working properly (by adding the 'on top' patch) now?  If so, do I put it into the same 'userChrome.css' file?

**Comment by MrOtherGuy** [comment]
Indeed, most of the time you won't need `window_control_placeholder_support.css` if you use the menubar patch, but I believe there are situations where you would still need the support style. Such as when you go to fullscreen mode (F11) then menubar isn't shown even if it is normally enabled.

If you you put contents of tabs_on_bottom.css directly to userChrome.css then I would also put contents of `window_control_placeholder_support.css` directly to userChrome.css. But if you are using `@import` statements to import tabs_on_bottom.css as a separate file then you should also import the support style. Nevertheless, you should load the support style first before tabs_on_bottom, and load the patch after tabs_on_bottom, to avoid conflicts.
